chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -120,6 +120,14 @@
         <partitions config:type="list">
           <partition>
             <create config:type="boolean">true</create>
+            <format config:type="boolean">true</format>
+           <filesystem config:type="symbol">vfat</filesystem>
+           <partition_id config:type="integer">259</partition_id>
+            <mount>/boot/efi</mount>
+            <size>128M</size>
+          </partition>
+          <partition>
+            <create config:type="boolean">true</create>
```

Comment:
It doesn't harm to create the partition unless you don't want to waste the disk space (I think we can reduce the 128M to much smaller values, 16 or 32MB should also work). 

I agree with you that the partition is not needed in non-EFI cases but I'm not sure how to detect that case. checking for arch==aarch64 is not enough since this is not a aarch64 specific issue. you have the same issue on x86 on systems that only boot via UEFI. (There are couple of bug reports about lack of UEFI support).